src/cli/test_command.zig

```diff
@@ -80,7 +80,7 @@ pub const CommandLineReporter = struct {
     last_dot: u32 = 0,
     summary: Summary = Summary{},
     prev_file: u64 = 0,
-    repeat_count: u32 = 1,
+    repeat_count: u32 = 0,
```

Comment:
Is this a drive-by bug fix? Can we make sure it doesn't break `--rerun-each`?

Reply:
Idk how that happened, I swear-